src/Components/Create.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from "react";
+import { PlusOutlined } from "@ant-design/icons";
+import { useNavigate } from "react-router-dom";
+import { Form, Input, DatePicker, InputNumber, Upload, notification, Rate } from "antd";
+import { ButtonDiv, StandardButton } from "./Style";
+
+
+const { RangePicker } = DatePicker;
+const { TextArea } = Input;
+
+type DateSetter = {
+    start: Date | undefined,
+    end: Date | undefined
+}
+
+const desc = ['terrible', 'bad', 'normal', 'good', 'wonderful'];
+
+const Create = () => {
+  const [title, setTitle] = useState("");
+  const [country, setCountry] = useState("");
+  const [location, setLocation] = useState("");
+  const [dateRange, setDateRange] = useState<DateSetter>({start:new Date(), end:new Date()}); //usestate utan egz tipust nativ date type 
+  const [description, setDescription] = useState("");
+  const [author, setAuthor] = useState("");
+  const [travelCompanions, setTravelCompanions] = useState<string | undefined>(
```

Comment:
Ez is mukodik, de ha ezt mindet a formban adod meg akkor lehet 1 stated is:

const [input, setInput] = useState({
{
title: '',
country: '',
location: '',
dateRange:
{
start: new Date( ),
end: new Date( )
},
...
}
)

Reply:
Itt majd nézzük meg, hogy ha 1 state-be rakom mindet, akkor a formban és a handleSubmit résznél,  hogyan tudom használni.